gtest-parallel

```diff
@@ -186,6 +190,11 @@ class FilterFormat:
     self.print_tests("INTERRUPTED TESTS", interruptions)
     self.out.flush_transient_output()
 
+    with zipfile.ZipFile(self.zip_file, 'w') as logs_zip:
```

Comment:
Please handle the case where self.passed = []
No need to write an empty zip file then.